bots/pr/src/main/java/org/openjdk/skara/bots/pr/CleanCommand.java

```diff
@@ -48,7 +48,11 @@ public boolean allowedInBody() {
     public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command, List<Comment> allComments, PrintWriter reply)
     {
         if (!censusInstance.isCommitter(command.user())) {
-            reply.println("Only OpenJDK [Committers](https://openjdk.org/bylaws#committer) can use the `/clean` command");
+            reply.println("To use the `/clean` command, you need to be a [Committer](https://openjdk.org/bylaws#committer) "
+                    + "in the OpenJDK [census](https://openjdk.org/census)"
+                    + " and your GitHub account needs to be linked with your OpenJDK username"
+                    + "([how to associate your GitHub account with your OpenJDK username]"
+                    + "(https://wiki.openjdk.org/display/skara#Skara-AssociatingyourGitHubaccountandyourOpenJDKusername)).");
```

Comment:
I don't think we need to change this message. It's not in the same category as the ones referenced in the bug. It was specifically the "contributor" reference that was misleading.